### ISSUE TITLE
Setting `V=0` does not reduce verbosity

### ISSUE THREAD

**Issue by nderjung** [unikraft/unikraft#676]
### Describe the bug

Setting the argument which alters build verbosity does not act as expected when set to `0`. Instead, the build verbosity increases.

### Steps to reproduce

Run a normal build using the Makefile system, adding `V=0` as an additional argument, e.g.:

```
make -C path/to/unikraft A=$(pwd) V=0
```

### Expected behavior

Verbosity should be reduced, only the build output should be printed, not the build instructions.

**Comment by ranfun** [comment]
I got the same error when I tried running the hello-world application on KVM-x86. I made some changes to the Makefile to deal with the problem, and it seems to work. Let me know if this works as a fix. 

The original Makefile - 
![image](https://user-images.githubusercontent.com/73271591/229063805-69ec2394-53fa-4605-ae70-1886e9208eb2.png)

The updated Makefile -
![1](https://user-images.githubusercontent.com/73271591/229063378-aeb85d3d-f708-41e2-ae22-f7c5b9936dbe.png)

Output with V=0 - 
![image](https://user-images.githubusercontent.com/73271591/229063523-61b20ee5-b7ae-45b7-8096-213ee174a4c6.png)

Output with V=1 -
![image](https://user-images.githubusercontent.com/73271591/229063677-e3c639de-6176-4bea-b06d-1a7cbead8bcb.png)
